src/bio/bio_internal.h

```diff
@@ -51,6 +53,83 @@ struct bio_dma_buffer {
 	ABT_mutex		 bdb_mutex;
 };
 
+#define BIO_PROTO_NVME_STATS_LIST					\
```

Comment:
@Michael-Hennecke I would appreciate your feedback on the list of smart infos that will be reported for each SSD below. Anything we are missing?